gh-120037: User site packages are being added when site module is enabled in _pth file (#120100)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2024-06-05-10-05-05.gh-issue-120037.QyZNce.rst

> Fix user site packages directory being added to `sys.path` when site module is enabled in `._pth` file. Patch by Tanzim Husain.

Reviewer: ```suggestion
Fix user site packages directory being enabled when a ``._pth`` file has been used. Patch by Tanzim Husain.
```

This is a ReST file, not Markdown, so the syntax is slightly different. This should work to allow the docs builds to pass.

**Edit** And simplified the text a little.
